### PR TITLE
Add criteria under <details> element

### PR DESCRIPTION
**Pull request by staltz**
For issue #22, this is just a first draft.

(The `<details>` element is handy to collapse/expand the criteria without cluttering the page)

@kof @mxstbr @piamancini?

### CODE REVIEW

File: README.md
Comment (kof) [inline README.md]: "to generate to profit" sounds like a typo?

Comment (kof) [reply README.md]: Also what about non-profit organizations but with funding?

Comment (staltz) [reply README.md]: I kind of feel like non-profits should be excluded because I would assume most open source maintainers are glad in letting their projects be used "for free" for good causes.

Comment (staltz) [reply README.md]: (and yes that was a typo)

File: README.md
Comment (kof) [inline README.md]: Q: Is it a good thing to call it sponsoring?  Isn't "investing" or "paying back" a better term?

Comment (kof) [reply README.md]: Sponsoring sounds to me like something you could do if you want to be a good person. I personally would try to express a stronger necessity than that.

File: README.md
Comment (kof) [reply README.md]: > I kind of feel like non-profits should be excluded because I would assume most open source maintainers are glad in letting their projects be used "for free" for good causes.

Sure, but the definition does mean that non-profits who invest into open source can't be on that list)

File: README.md
Comment (staltz) [reply README.md]: I kind of prefer sponsorship, probably because my ex-employer Futurice used it: https://spiceprogram.org/

I see investment as expectancy of ROI, in other words getting in channel X more than you give in channel X (where X here is usually money). I see sponsorship as giving in one kind (e.g. money) and receiving in some other kind (e.g. marketing).

Comment (kof) [reply README.md]: Yeah, though the thing is once company starts using a project, they already got something, so ROI  happened before they even started investing. So maybe paying back is the right term?

Comment (staltz) [reply README.md]: Other possible word: support.

File: README.md
Comment (staltz) [reply README.md]: Yeah, they can't. Should they? (given the "goals" paragraph above)

Comment (kof) [reply README.md]: The thing is I know a company who is non-profit and who does invest into open source and who tries to hire people and it would be unfair to them to not be able to get into that list.

File: README.md
Comment (kof) [reply README.md]: Yeah , support sounds ok to me too.

File: README.md
Comment (staltz) [reply README.md]: Well, maybe. I see this list as "companies that make profit but pay back". That said, we could have a "non-profits" section.

Comment (kof) [reply README.md]: This or it simply should't matter how much money a company earns or none. Also think of state organizations.

Comment (kof) [reply README.md]: Also think of all the startups, who most of the time just burning investors money. And yet open source is not considered as one of their expenses.

Comment (kof) [reply README.md]: There should be absolutely no problem for a startup with 500k seed investment to give 1k or something per year away for open source.

File: README.md
Comment (abitrolly) [inline README.md]: ...highlight companies who pay back to open source projects that are helping them to save time and money. Open source maintainers are often an invisible and underfunded workforce behind our informational infrastructure. With this list we would like to encourage more companies to be aware of that and play active role in supporting our common grounds.

Comment (staltz) [reply README.md]: Sounds better than what I wrote. :)

**Comment by staltz** [comment]
Updated it, see the last 2 commits in the PR.

**Comment by kof** [comment]
Should we address non-profits, state organizations and startups without a revenue in a separate PR?

**Comment by staltz** [comment]
Yes, I think so, specially because we don't have any non-profit yet in the list (except Mozilla, but they are part corporation, part non-profit) or in PRs.

**Comment by kof** [comment]
Yeah but startups without revenue ... its guaranted an issue.

**Comment by kof** [comment]
Though startups aim to have revenue at some point, so maybe we can at least have a wording that includes intention as well.

**Comment by staltz** [comment]
Yes, startups are *made* to generate profit even though they might not be currently.

> COMPANY: a legal entity made to generate profit.

**Comment by kof** [comment]
right, then this is covered

**Review by kof — APPROVED** [review]

**Comment by staltz** [comment]
I'll wait a bit more for other reviews before merging.

File: README.md
Comment (mxstbr) [inline README.md]: I would put this outside of the `details` so the folks reading have some context on the companies included in the list?

File: README.md
Comment (mxstbr) [inline README.md:19]: I like "open source projects that they don't control" or "external open source projects" more than "third-party"? Maybe just a nitpick, so feel free to ignore.

**Review by mxstbr — CHANGES_REQUESTED** [review]

File: README.md
Comment (staltz) [reply README.md]: Good point.

**Comment by staltz** [comment]
Should be good for merging now, I guess.

**Pull request merged**
